client/src/stores/UIStore.js

```diff
@@ -7,12 +7,22 @@ class UIStore {
     makeObservable(this, {
       sidebarState: observable,
       toggleSidebarState: action,
+      closeSidebar: action,
+      openSidebar: action,
     });
   }
 
   toggleSidebarState = () => {
     this.sidebarState = !this.sidebarState;
   };
+
+  closeSidebar = () => {
+    this.sidebarState = false;
+  };
+
+  openSidebar = () => {
```

Comment:
love the three functions to set a boolean

Reply:
it’s called convenience